addons/website_slides/controllers/main.py

```diff
@@ -549,6 +549,56 @@ def slide_channel_tag_search_read(self, fields, domain):
             'can_create': can_create,
         }
 
+    @http.route(['/slides/channel/tag/group/search_read'], type='json', auth='user', methods=['POST'], website=True)
+    def slide_channel_tag_group_search_read(self, fields, domain):
+        can_create = request.env['slide.channel.tag.group'].check_access_rights('create', raise_exception=False)
+        return {
+            'read_results': request.env['slide.channel.tag.group'].search_read(domain, fields),
+            'can_create': can_create,
+        }
+
+    @http.route('/slides/channel/tag/add', type='json', auth='user', methods=['POST'], website=True)
+    def slide_channel_tag_add(self, **post):
```

Comment:
Looking at the code, it seems we could avoid using kwargs and give the required parameters: channel_id, tag_id, group_id, and explain that tag_id is either an existing ID, either the value returned by select2. Same for group_id .

That way code becomes more explicit. What do you think ?

Reply:
Sure. I changed the arguments and updated the docstring accordingly.